libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -99,7 +100,7 @@ export class RadioGroupComponent implements AfterContentInit, ControlValueAccess
   private _disabled = false;
   private _items: string[] | any[] = [];
   private _onChangeCallback: (value: any) => void = () => {};
-  private _onTouched: Function;
+  private _onTouched = noop;
```

Comment:
Why the extra abstraction using noop from RxJS compared to just inlining a noop function?
And why differ from the implementation in dropdown?
```suggestion
  private _onTouched = () => {};
```

Reply:
I think a named function like `noop` is more explicit about "nothing is going to happen" (even though it's a minor difference in this case). If it's `()  => {}` in the dropdown implementation we should go with that here as well.